roseus/euslisp/actionlib.l

```diff
@@ -93,20 +102,24 @@
    (cond
     (groupname
      (ros::subscribe (format nil "~A/status" name-space)
-                     actionlib_msgs::GoalStatusArray #'send self :goal-status-cb 1 :groupname groupname)
+                     actionlib_msgs::GoalStatusArray #'send self :goal-status-cb 8 :groupname groupname)
      (ros::subscribe (format nil "~A/result" name-space)
-                     action-result-class #'send self :action-result-cb 1 :groupname groupname)
+                     action-result-class #'send self :action-result-cb 8 :groupname groupname)
      (ros::subscribe (format nil "~A/feedback" name-space)
-                     action-feedback-class #'send self :action-feedback-cb 1 :groupname groupname))
+                     action-feedback-class #'send self :action-feedback-cb 8 :groupname groupname))
     (t
      (ros::subscribe (format nil "~A/status" name-space)
-                     actionlib_msgs::GoalStatusArray #'send self :goal-status-cb 1)
+                     actionlib_msgs::GoalStatusArray #'send self :goal-status-cb 8)
      (ros::subscribe (format nil "~A/result" name-space)
-                     action-result-class #'send self :action-result-cb 1)
+                     action-result-class #'send self :action-result-cb 8)
      (ros::subscribe (format nil "~A/feedback" name-space)
-                     action-feedback-class #'send self :action-feedback-cb 1)))
+                     action-feedback-class #'send self :action-feedback-cb 8)))
    (setq simple-state ros::*simple-goal-state-done*)
+   (push self ros::*simple-action-client-list*)
    self)
+  (:dispose
+   ()
+   (setq ros::*simple-action-client-list* (remove self ros::*simple-action-client-list*)))
```

Comment:
since we push `simple-action-client` to global list, it will never disposed.... need FIX